New Excercises (#30)
Akuli/python-tutorial: basics/handy-stuff-strings.md

    print(message, "!!!")
    print(message, "!!!")
    ```

> 3. Make a program to take the input from the user and check if it is a palindrome.

Reviewer: Nit:  "take the input" feels a bit weird to me.

```suggestion
3. Make a program that asks the user to type a string and checks if it is a palindrome.
```

    print(message, "!!!")
    print(message, "!!!")
    ```
3. Make a program to take the input from the user and check if it is a palindrome.<br>

>    (Hint: Google how to reverse a string)

Reviewer: I was going to add this on the answer page, but this might be a better place for it:

```suggestion
   (Hint: A string is a palindrome if it is the same when reversed. Google how to reverse a string.)
```
